website/src/components/versionBlock/versionBlock.cy.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react'
+import VersionBlock from './index.js'
+// import { mount } from '@cypress/react'
+// import VersionContextProvider from '../../stores/VersionContext.js'
+
+describe('Test VersionBlock component', () => {
+  it('Should render same text passed into title prop', () => {
+    cy.mount(<VersionBlock>Versioned content</VersionBlock>)
+    cy.contains('Versioned content').should('be.visible')
+  })
+  // Developer note:
+  // Need to figure out how to mount muliple components
+  // -----------------
+  // const testData = [
+  //   {
+  //     desc: 'Should render content if current version >= firstVersion',
+  //     version: "0.21",
+  //     shouldShowContent: true
+  //   },
+  //   {
+  //     desc: 'Should not render content if current version less than firstVersion',
+  //     version: "0.20",
+  //     shouldShowContent: false
+  //   },
+  // ]
+  // testData.forEach((testCase) => {
+  //   it(testCase.desc, () => {
+  //     localStorage.setItem('dbtVersion', testCase.version)
+  //     const firstVersion = "0.21"
+
+  //     mount(
+  //       <VersionContextProvider>
+  //         <VersionBlock firstVersion={firstVersion}>
+  //           Current version
+  //         </VersionBlock>
+  //       </VersionContextProvider>
+  //     )
+  //     cy.contains('Current version').should('be.visible')
+  //   })
+  // })
+})
```

Comment:
I'm open to trying to solve this with you for this PR. I tried using the [cypress-each](https://github.com/bahmutov/cypress-each) package which is supposed to make this feature available, however i'm receiving an error. 

Even if we can't use the `it.each`, maybe we can write the two tests separately for this.

Reply:
Here's the code from what we just went over!

We're adjusting the import for the context to:
`import VersionContext from '../../stores/VersionContext.js'` rather than `VersionContextProvider`

Then the two tests can be broken out into individual tests:
```
it('Should render content if current version >= firstVersion', () => {
    cy.mount(
      <VersionContext.Provider value={{ version: '1.0'}}>
        <VersionBlock firstVersion={'0.9'}>
          Current version
        </VersionBlock>
      </VersionContext.Provider>
    )
    cy.contains('Current version').should('exist')
  })
  
  it('Should not render content if current version less than firstVersion', () => {
    cy.mount(
      <VersionContext.Provider value={{ version: '1.0'}}>
        <VersionBlock firstVersion={'1.1'}>
          Current version
        </VersionBlock>
      </VersionContext.Provider>
    )
    cy.contains('Current version').should('not.exist')
  })
```

The `<VersionContext.Provider value={{ version: '1.0'}}>` line is emulating what the version is on the site. That version value is compared to the `firstVersion` prop being passed into the VersionBlock component.

Reply:
Thank you ! This makes sense !